catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/CswEndpoint.java

```diff
@@ -929,6 +932,17 @@ private Element loadDocElementFromResourcePath(String resourcePath) throws CswEx
         URL recordUrl = context.getBundle()
                 .getResource(resourcePath);
 
+        if (recordUrl == null) {
+            Optional<Bundle> bundle = Stream.of(context.getBundles())
+                    .filter(b -> b.getSymbolicName()
+                            .equals("spatial-csw-schema-bindings"))
```

Comment:
Instead of using the symbolic name, can we use a class that is exposed by that bundle?  That may be cleaner and prevent future issues if that project is refactored.